docs/test_scenarios.rst

```diff
@@ -0,0 +1,729 @@
+              created entry and passing an invalid value for the ``--vault``
+              option.
+:steps: Run ``rho profile clear --name <name> --vault <invalidvault>``
+:expectedresults: The profile entry is not removed and the ``profiles`` file is
+                  not updated.
+:bug: https://github.com/quipucords/rho/issues/132
+
+Clear multiple profile entries with vault (negative)
+----------------------------------------------------
+
+:description: Clear multiple profile entries using the ``--all`` option and
+              passing an invalid value for the ``--vault`` option.
+:steps: Run ``rho profile clear --all --vault <invalidvault>``
+:expectedresults: All profile entries are not removed and the ``profiles`` file
+                  is not deleted.
+:bug: https://github.com/quipucords/rho/issues/132
+
+rho profile edit
+================
+
+Edit profile hosts
+------------------
+
+:description: Edit the hosts of an profile entry.
+:steps: Run ``rho profile edit --name <name> --hosts <newhosts>``
+:expectedresults: The profile hosts must be updated and the ``profiles``
+                  file must be updated.
+
+Edit profile sshport
+--------------------
+
+:description: Edit the sshport of an profile entry.
+:steps: Run ``rho profile edit --name <name> --sshport <newsshport>``
+:expectedresults: The profile sshport must be updated and the ``profiles``
+                  file must be updated.
+
+Edit profile auth
+-----------------
+
+:description: Edit the auth of an profile entry.
+:steps: Run ``rho profile edit --name <name> --auth <newauth>``
+:expectedresults: The profile auth must be updated and the ``profiles``
+                  file must be updated.
+
+Edit profile hosts with vault
+-----------------------------
+
+:description: Edit the hosts of an profile entry passing the ``--vault``
+              option.
+:steps: Run ``rho profile edit --name <name> --hosts <newhosts> --vault
+        <vault>``
+:expectedresults: The profile hosts must be updated and the ``profiles``
+                  file must be updated.
+
+Edit profile sshport with vault
+-------------------------------
+
+:description: Edit the sshport of an profile entry passing the ``--vault``
+              option.
+:steps: Run ``rho profile edit --name <name> --sshport <newsshport> --vault
+        <vault>``
+:expectedresults: The profile sshport must be updated and the ``profiles``
+                  file must be updated.
+
+Edit profile auth with vault
+----------------------------
+
+:description: Edit the auth of an profile entry passing the ``--vault``
+              option.
+:steps: Run ``rho profile edit --name <name> --auth <newauth> --vault
+        <vault>``
+:expectedresults: The profile auth must be updated and the ``profiles``
+                  file must be updated.
+
+Edit profile hosts (negative)
+-----------------------------
+
+:description: Try to edit the hosts of a not created n profile entry.
```

Comment:
several entries of `not created n profile entry` that should have the `n ` removed.